src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```
@@ -52,6 +52,7 @@ import {
 import { getTableNodes, getTableEditingPlugins } from './plugins/table-plugin';
 import { getImageNode, imageCache } from './plugins/image/node';
 import { EditorUiType } from '../types';
+import { Selection } from 'prosemirror-state';
```

ESLint

[error] 'prosemirror-state' import is duplicated.

(no-duplicate-imports)